dist/src/test/java/io/camunda/webapps/controllers/BackupControllerTest.java

```diff
@@ -166,6 +166,21 @@ public void shouldGetBackups() {
     assertThat(response.getBody()).isEqualTo(List.of(EXPECTED_INFO));
   }
 
+  @Test
+  public void shouldGetBackupsWithNullParameters() {
+    when(backupService.getBackups(anyBoolean(), any()))
```

Comment:
```suggestion
    when(backupService.getBackups(eq(null), eq(null))
```
:wrench: if there's another matcher with null it's even better